app/(candidate)/dashboard/pro-sign-up/committee-check/components/CommitteeCheckPage.js

```diff
@@ -0,0 +1,216 @@
+'use client';
+import { FocusedExperienceWrapper } from 'app/(candidate)/dashboard/shared/FocusedExperienceWrapper';
+import H1 from '@shared/typography/H1';
+import Body2 from '@shared/typography/Body2';
+import Link from 'next/link';
+import SecondaryButton from '@shared/buttons/SecondaryButton';
+import PrimaryButton from '@shared/buttons/PrimaryButton';
+import TextField from '@shared/inputs/TextField';
+import React, { useEffect, useRef, useState } from 'react';
+import gpFetch from 'gpApi/gpFetch';
+import gpApi from 'gpApi';
+import { updateCampaign } from 'app/(candidate)/onboarding/shared/ajaxActions';
+import { useRouter } from 'next/navigation';
+import { AsyncValidationIcon } from 'app/(candidate)/dashboard/shared/AsyncValidationIcon';
+import { HiddenFileUploadInput } from '@shared/inputs/HiddenFileUploadInput';
+import {
+  EIN_PATTERN_FULL,
+  EinCheckInput,
+} from 'app/(candidate)/dashboard/pro-sign-up/committee-check/components/EinCheckInput';
```

Comment:
I think because I told it not not use relative paths because you had a comment in another PR that was saying you didn't like seeing relative paths? 🤷

Reply:
Generally, we do relative until three "up directory" "../../../"